adapters/src/main/java/org/jboss/jca/adapters/AdaptersLogger.java

```diff
@@ -323,4 +323,13 @@
    @LogMessage(level = WARN)
    @Message(id = 30060, value = "Error checking state")
    public void errorCheckingState(@Cause Throwable t);
+
+   /**
+    * Auto commit reset
+    * @param msg The message
+    * @param t The throwable
+    */
+   @LogMessage(level = WARN)
+   @Message(id = 30061, value = "Error resetting auto-commit for: %s")
+   public void autoCommitReset(String msg, @Cause Throwable t);
```

Comment:
chane method name in "errorResettingAutoCommit". It's more readable and follow the same rule of other methods of the class